.github/workflows/release-main.yml

```
@@ -118,88 +115,10 @@ jobs:
-      channel: stable
-      candidate_sha: ${{ needs.semantic-release.outputs.release_sha }}
-      base_tag: ${{ needs.prepare-release-notification.outputs.base_tag }}
-      current_version: ${{ needs.semantic-release.outputs.release_tag }}
-      details_url: ${{ needs.prepare-release-notification.outputs.release_url }}
-      compare_url: ${{ needs.prepare-release-notification.outputs.compare_url }}
-      mention_text: ${{ vars.DISCORD_STABLE_MENTIONS }}
-      predicted_version: ${{ needs.prepare-release-notification.outputs.release_version }}
-
   backport-release:
     name: Backport Release Commit to Develop
     if: ${{ needs.semantic-release.outputs.released == 'true' && needs.publish-release.result == 'success' }}
```

actionlint (1.7.12)

[error] property "publish-release" is not defined in object type {semantic-release: {outputs: {release_sha: string; release_tag: string; released: string}; result: string}}

(expression)